src/DIRAC/WorkloadManagementSystem/DB/JobDB.py

```diff
@@ -2075,7 +2074,7 @@ def removeInfoFromHeartBeatLogging(self, status, delTime, maxLines):
             return ret
         delTime = ret["Value"]
 
-        self.log.verbose("Removing HeartBeatLogginInfo for", "%r %r %r" % (status, delTime, maxLines))
+        self.log.verbose("Removing HeartBeatLogginInfo for", f"{status} {delTime} {maxLines}")
```

Comment:
Do you want to keep the use if `repr` here?
```suggestion
        self.log.verbose("Removing HeartBeatLogginInfo for", f"{status!r} {delTime!r} {maxLines!r}")
```